src/test/java/api/endpoints/LaunchesApi.java

```diff
@@ -0,0 +1,6 @@
+package api.endpoints;
+
+public class LaunchesApi {
```

Comment:
I would suggest you to delete this class and move the URL to the APIConstants.java class under the 'utils' folder along with other constants.

Reply:
Fixed